tests/contrib/rename_image_test.sh

```diff
@@ -0,0 +1,42 @@
+# Copyright 2015 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#!/bin/bash
+
+function extract_image_name () {
+    # Extracts the image name (repo:tag) from the tarball without running it
+    # Does not require docker to be installed
+
+    tar -xf "$1" "manifest.json"
```

Comment:
create a var somewhere with a meaningful name and assign it as value $1, so that you dont have to use $1 enlsewhere (same for $2)

Reply:
Good call. Changed.